pkg/controller/component/exporter_rbac.go

```diff
@@ -0,0 +1,106 @@
+package component
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/vesoft-inc/nebula-operator/apis/apps/v1alpha1"
+)
+
+const (
+	exporterRoleName           = "exporter-role"
+	exporterRoleBindingName    = "exporter-rolebinding"
+	exporterServiceAccountName = "exporter-sa"
+)
+
+func (e *nebulaExporter) checkExporterRBAC(nc *v1alpha1.NebulaCluster) error {
+	if err := e.createServiceAccount(nc); err != nil {
```

Comment:
The rbac of operaotor seems not to be updated, because you need new permissions?